public/resources/remix/style.less

```diff
@@ -16,6 +16,7 @@
   top: 0 !important;
   right: 0 !important;
   font-family: "Fira Medium", "Helvetica Neue", sans-serif !important;
+  z-index: 999 !important;
```

Comment:
Brackets uses `2000000` for the highlights it puts in the preview.  I'd do that too, since we know it works well.